libs/text-splitters/tests/unit_tests/test_text_splitters.py

```diff
@@ -1619,6 +1619,37 @@ def test_happy_path_splitting_based_on_header_with_whitespace_chars() -> None:
     assert docs[2].metadata["Header 2"] == "Baz"
 
 
+@pytest.mark.requires("lxml")
+@pytest.mark.requires("bs4")
+def test_section_splitter_accepts_a_relative_path() -> None:
+    html_string = """<html><body><p>Foo</p></body></html>"""
+    test_file = Path("tests/test_data/test_splitter.xslt")
+    assert test_file.is_file()
+
+    sec_splitter = HTMLSectionSplitter(
+        headers_to_split_on=[("h1", "Header 1"), ("h2", "Header 2")],
+        xslt_path=test_file.as_posix(),
```

Comment:
out of curiosity why do we need to add as_posix here and in 1647

Reply:
Hmmm, I guess we don't need to anymore. Initially, we were assigning either a string or a `Path` to `xslt_path`/`self.xslt_path` so I used `.as_posix()` to give us a cross-platform compatible string. Now we don't have the default arg as a string, it could always be an instance of `Path` and we can simplify our logic. I'll push up an improvement, thanks for the review :)